src/regression.js

```diff
@@ -1,41 +1,6 @@
-const DEFAULT_OPTIONS = { order: 2, precision: 2, period: null };
+const { round, deriveDataProperties } = require('./utils');
```

Comment:
CI tests complained about this file being too big, and that there was duplicate code.
Moved that to `utils.js`